File: selectionSort.cpp
#include <iostream>
using namespace std;

int main()
{
    int arr[] = {8,7,3,4,0,9,2,6};
    for (int i = 0; i < 8; i++)
    {
        int temp = arr[i];
        for (int j = i+1; j < 8; j++)
        {
            if(arr[j]<temp){
                arr[i] = arr[j];
                arr[j] = temp;
                temp = arr[i];
            }
                // break;
        }
    }

    for (int i = 0; i < 8; i++)
    {
        cout << *(arr + i) << endl;
    }
}

File: selectionSort.py
arr = [4,5,2,9,6,3,8,1,2]


for i in range(1,len(arr)):
    j = i-1
    temp = arr[i]
    while(j>=0):
        if(arr[j]>temp):
            arr[j+1] = arr[j]
            j-=1
        else:
            break
    arr[j+1] = temp

for i in arr:
    print(i)
